Makefile

```diff
@@ -31,5 +31,17 @@ linter:
 pre-commit: fmt linter test
 
 test: build
+	@\
+	if [ ! -f "./target/wasm32-unknown-unknown/release/fungible_token.opt.wasm" ]; then\
+	    curl -L\
+	        "https://github.com/gear-dapps/fungible-token/releases/download/build/fungible_token.opt.wasm"\
+	        -o "./target/wasm32-unknown-unknown/release/fungible_token.opt.wasm";\
```

Comment:
All WASMs from `./target/wasm32-unknown-unknown/release` directory will be uploaded to GitHub Releases after CI runs, so I'd recommend using a different path for external WASMs (e.g. `./target/fungible_token.opt.wasm` as it has been done in other repos here).